### PR TITLE
Give more detailed explanation of `args` argument to `rcmdcheck`.

### PR DESCRIPTION
**Pull request by rorynolan**
This is the PR arising from #124.
I also did some spell checking :-)

Thanks for the package and all of your work. Every working day I use things that you built :-)

### CODE REVIEW

File: R/package.R [hunk 1]
```diff
@@ -12,34 +12,39 @@ NULL
...
+#'   no timeout. If the check is timed out, that is added as an extra error to
+#'   the result object.
+#' @param error_on Whether to throw an error on `R CMD check` failures. Note
+#'   that the check is always completed (unless a timeout happens), and the
+#'   error is only thrown after completion. If `"never"`, then no errors are
+#'   thrown. If `"error"`, then only `ERROR` failures generate errors. If
+#'   `"warning"`, then `WARNING` failures generate errors as well. If `"note"`,
+#'   then any check failure generated an error.
+#' @return An S3 object (list) with fields `errors`, `warnings` and `notes`.
+#'   These are all character vectors containing the output for the failed check.
```
Comment (gaborcsardi) [inline R/package.R:47]: Thanks! Do we actually need all these other changes? It is hard for me to follow what has changed here in addition to `args`. Thanks.

Comment (rorynolan) [reply R/package.R:47]: The only thing I changed in other files are spelling corrections.
In `package.R`, the only things I changed are `@param args` and `@param build_args`; I felt that both needed the same sort of explanation. It's a little bit long, but I think it's better than the alternative, which was confusing. Feel free to edit/shorten/reject I really don't mind and you probably know best.

Comment (gaborcsardi) [reply R/package.R:47]: Thanks! My guess is that your IDE re-formatted the whole roxygen block, unfortunately....

Comment (rorynolan) [reply R/package.R]: Yes that is what happened. I've fixed it now, reverting the other arguments back to how they were before. Is that showning up for you now?

Comment (gaborcsardi) [reply R/package.R]: Yep, looks great, thanks!

**Comment by gaborcsardi** [comment]
Thanks much!

**Comment by rorynolan** [comment]
You're welcome! I have a final question. Do you think `devtools::check()` should `@inheritParams rcmdcheck::rcmdcheck`? Seems to me like it should. Right now, `devtools::check()` has the old confusing `args`. Should I submit a PR there?

**Comment by gaborcsardi** [comment]
> Should I submit a PR there?

Sure. IDK about inheriting the args, or just adding a short explanation, up to the devtools maintainer I guess. :) Thanks!

File: R/package.R [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
 #' Run R CMD check from R and Capture Results
 #'
-#' Run R CMD check form R programatically, and capture the results of the
+#' Run R CMD check form R programmatically, and capture the results of the
```
Comment (jimhester) [inline R/package.R:4]: typo here, from, not *form*.

**Pull request merged**